RELEASE.md

```diff
@@ -6,6 +6,10 @@ Please follow the established format:
 - Include the ID number for the related PR (or PRs) in parentheses
 -->
 
+# Release 5.0.2
+## Bug fixes and other changes
+- Upgrade to dart-scss, and update the recommended Node version to v16. (#774)
```

Comment:
```suggestion
- Upgrade to `dart-scss` and update the recommended Node version to v16. (#1026)
```

Reply:
FYI we use the PR number in the release note, not the issue number :)

Reply:
ah thanks for letting me know 👍